components/Modules/parts/ModuleTitle.tsx

```
@@ -0,0 +1,135 @@
+          >
+            <Check className="h-5 w-5 text-green-500" />
+          </Button>
+          <Button variant="ghost" size="icon" onClick={cancelTitleEdit}>
+            <X className="h-5 w-5 text-red-500" />
+          </Button>
+        </div>
+      ) : (
+        <h1
+          className="text-xl cursor-pointer hover:bg-muted/50 px-2 py-1 rounded"
+          onClick={() => setIsEditingTitle(true)}
+          tabIndex={0}
```

Biome (1.9.4)

[error] The HTML element h1 is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)